components/offices/testimonialPanel.tsx

```diff
@@ -1,21 +1,19 @@
-import { TinaMarkdown, TinaMarkdownContent } from "tinacms/dist/rich-text";
-
 type TestimonialPanelProps = {
-  testimonial: {
-    body?: TinaMarkdownContent;
-    name?: string;
-    company?: string;
-  };
+  body?: string;
+  name?: string;
+  company?: string;
 };
 
-const TestimonialPanel = ({ testimonial }: TestimonialPanelProps) => {
+const TestimonialPanel = ({ props }) => {
+  if (!props) return <></>;
+  const { body, name, company }: TestimonialPanelProps = JSON.parse(props);
```

Comment:
JSON.parse shouldn't be necessary - bad code smell

Reply:
Fixed - Now passing a name as a string to retrieve the testimonial.